SPECS/gd/gd.spec

```diff
@@ -38,6 +38,7 @@ BuildRequires:  fontconfig-devel
 # needed for tests
 BuildRequires:  libjpeg-devel
 BuildRequires:  libpng-devel
+BuildRequires:  fontawesome-fonts
```

Comment:
If this is only needed for tests, could you put it behind a `%if %{with_check}` conditional?